src/book.js

```diff
@@ -434,22 +461,30 @@ export class Book {
     return [newPdf, embeddedPages];
   }
 
+  async collatePages(pdfA, pdfB) {
```

Comment:
I think interleave might be clearer than collate. We might want to pull all the pure PDF manipulation stuff out into a helper class like:

```javascript
class PDFHelper {
    static interleave(pdfA, pdfB){
    // code here
    };
    
    static async embedPagesInNewPdf{
    // code here
    };
}

PDFHelper.interleave(a, b);
```

Reply:
yeah you might be right - though I'd just make it a utils file, rather than a class.

Reply:
I guess a class only matters if you want to make some of the methods private, then again I guess you can just not export them if it's a file. Potato Potato.